config/locales/forms/renews/en.yml

```diff
@@ -0,0 +1,9 @@
+en:
+  waste_carriers_engine:
+    renews:
+      already_renewed:
+        heading: "That registration has already been renewed"
+        paragraph1: "Our records show that registration %{reg_identifier} has already been renewed."
+        paragraph2_html: "If you need to register other waste exemptions you should <a href=\"%{link_to_new_registration}\">make a new registration.</a>"
```

Comment:
Waste exemptions? Think we may need to double-check this

Reply:
Ah! I copy-pasted from the wireframe and didn't notice! Will ask for an updated text!